metrics-datadog/src/main/java/org/coursera/metrics/datadog/DatadogReporter.java

```diff
@@ -74,36 +74,27 @@ public void report(SortedMap<String, Gauge> gauges,
                      SortedMap<String, Meter> meters,
                      SortedMap<String, Timer> timers) {
     final long timestamp = clock.getTime() / 1000;
-
```

Comment:
Tag generation needs to be moved deeper as the reporter is generating metric names deeper inside (where it loops over a metric to generate P99, P75, etc.